packages/default/scss/tabstrip/_layout.scss

```diff
@@ -71,6 +71,13 @@
             align-items: stretch;
             justify-items: stretch;
             outline: 0;
+
+            // Active state
+            &:active,
+            &.k-active,
+            &.k-selected {
+                font-weight: var( --kendo-tabstrip-item-selected-font-weight, #{$kendo-tabstrip-item-selected-font-weight} );
```

Comment:
The font weight here should be set in the following way:
font-weight: $kendo-tabstrip-item-selected-font-weight;